packages/trpc/server/routers/viewer/teams/inviteMember.handler.ts

```diff
@@ -112,13 +113,36 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
     }
     // inform user of membership by email
     if (input.sendEmailInvitation && ctx?.user?.name && team?.name) {
+      const inviteTeamOptions = {
+        joinLink: `${WEBAPP_URL}/auth/login?callbackUrl=/settings/teams`,
+        isCalcomMember: true,
+      };
+      /**
+       * Here we want to redirect to a differnt place if onboarding has been completed or not. This prevents the flash of going to teams -> Then to onboarding - also show a differnt email template.
+       * This only changes if the user is a CAL user and has not completed onboarding and has no password
+       */
+      if (!invitee.completedOnboarding && !invitee.password && invitee.identityProvider === "CAL") {
+        const token = randomBytes(32).toString("hex");
+        await prisma.verificationToken.create({
+          data: {
+            identifier: input.usernameOrEmail,
+            token,
+            expires: new Date(new Date().setHours(168)), // +1 week
+          },
+        });
```

Comment:
This fires if a User has an existing team invite and gets invited to another team. 

A user can exist but might not have completed onboarding or have a password set. So we generate a signup link + callback to getting-started to ensure the user doesnt get any flashes of the wrong page